collect/collect.go

```diff
@@ -196,38 +199,21 @@ func (i *InMemCollector) AddSpanFromPeer(sp *types.Span) {
 // block is the only place we are allowed to modify any running data
 // structures.
 func (i *InMemCollector) collect() {
-	// make sure we send all pending traces when we finish
```

Comment:
What happens now on shut down? Will pending traces still be sent?

Reply:
Pending traces should still be sent as the cache will be iterated over when we call `Stop` on the collector down [here](https://github.com/honeycombio/samproxy/blob/b5589bf57cd9b14a0e6d8e36326985b7f9796a92/collect/collect.go#L435)